### ISSUE TITLE
Support external resource packs

### ISSUE THREAD

**Issue by terrarier2111** [Lea-fish/Leafish#27]
This is also a nice to have feature, but low prio

**Comment by Z2Up1UwcaYOyZq** [comment]
shout out to #47, because it needs to be done before this one

**Comment by iceiix** [comment]
> A nice bonus would be to eventually ship out of the box with a resource pack with some open-source license. 

Looked into open source texture packs a while back. TrueCraft uses the Pixeludi pack: https://github.com/ddevault/TrueCraft/tree/master/TrueCraft.Client/Content - but they only support up to 1.7.3 beta (2011), and in the older texture atlas format. Before Pixeludi they were using https://github.com/deathcap/ProgrammerArt which supports blocks and items up to 1.9 (in either newer "texture packs" and "resource packs" zips, or the old atlas pngs) but is not the most visually appealing, mostly only suitable as a placeholder. A fully complete, updated, and decently looking OSS resource pack would be very nice indeed, I'm looking for one too (but have yet to find any).

**Comment by PureTryOut** [comment]
Not the most visual appealing indeed, I'm not a fan at least haha. Preferably something is used that keeps the original feel of the game.

It seems most resourcepacks out there are proprietary or at least not freely licensed, quite disappointing.
